test/powershell/Language/Scripting/Debugging/DebuggerScriptTests.Tests.ps1

```diff
@@ -290,17 +290,13 @@ Describe "Unit tests for various script breakpoints" -Tags "CI" {
         {
             & $command
 
-	        It "Script should fail to verify exception" {
-		        $false | Should Be $true
-	        }
+            throw "No Exception!"
         }
         catch
         {
-            $type = $_.Exception.GetType().Name
-
-	        It "Script failed expected exception" {
-		        $type | Should Be $exception
-	        }
+            It "Script failed expected exception" {
+                $_.Exception.GetType().Name | Should Be $exception

```

Comment:
I see what is happening. $exception is point to internal types, and thus Pester cannot resolve. #Closed